src/login/AzureLoginHelper.ts

```diff
@@ -171,14 +139,10 @@ export class AzureLoginHelper {
 		});
 	}
 
-	private async sendLoginTelemetry(context: IActionContext, trigger: LoginTrigger, path: CodePath, cloud: string, outcome: string, message?: string, includeSubscriptions?: boolean) {
+	private async sendLoginTelemetry(context: IActionContext, properties: { trigger: LoginTrigger, codePath: CodePath, environmentName: string, outcome: string, message?: string }, includeSubscriptions?: boolean) {
```

Comment:
I've been meaning to do this for a while: it's bad practice to have a bunch of strings in sequence for function arguments because they can be easily swapped so this makes it more robust & the code is cleaner.

Reply:
Other than this are there any substantive changes or is it just splitting it out into multiple files?

Reply:
> Other than this are there any substantive changes or is it just splitting it out into multiple files?

Nope these were the only substantive changes. It's mainly just moving a lot of code around